.circleci/config.yml

```diff
@@ -80,6 +84,29 @@ jobs:
       SIGNAC_VERSION: "signac==1.8.0"
       PYTHON: python
 
+  test-install-pip-python-311: &test-install-pip
```

Comment:
I'd be fine with only keeping one of these jobs, and only running 3.11 instead of 3.10.